packages/prosess-vilkar-opptjening-oms/src/OpptjeningVilkarProsessIndex.tsx

```diff
@@ -58,8 +58,8 @@ const OpptjeningVilkarProsessIndex = ({
 
   const [activeVilkår] = vilkar;
   const skalBrukeSidemeny = activeVilkår.perioder.length > 1;
-  const perioder = activeVilkår.perioder.filter(periode => visAllePerioder || periode.vurdersIBehandlingen);
-
+  const perioder = activeVilkår.perioder.filter(periode => visAllePerioder && !periode.vurdersIBehandlingen
+    || !visAllePerioder && periode.vurdersIBehandlingen);
```

Comment:
Eg foretrekker å dele inn med paranteser slik at det er enkelere å lese korleis den vurderes.
```suggestion
  const perioder = activeVilkår.perioder.filter(periode => (visAllePerioder && !periode.vurdersIBehandlingen)
    || (!visAllePerioder && periode.vurdersIBehandlingen));
```